litex/soc/software/bios/boot.c

```diff
@@ -395,14 +511,14 @@ static void netboot_from_bin(const char * filename, unsigned int ip, unsigned sh
 void netboot(void)
 {
 	unsigned int ip;
-
-
-	printf("Booting from network...\n");
-	printf("Local IP : %d.%d.%d.%d\n", LOCALIP1, LOCALIP2, LOCALIP3, LOCALIP4);
-	printf("Remote IP: %d.%d.%d.%d\n", REMOTEIP1, REMOTEIP2, REMOTEIP3, REMOTEIP4);
-
-	ip = IPTOINT(REMOTEIP1, REMOTEIP2, REMOTEIP3, REMOTEIP4);
-	udp_start(macadr, IPTOINT(LOCALIP1, LOCALIP2, LOCALIP3, LOCALIP4));
+	
+        printf("Booting from network...\n"); 
+	
+	printf("Local IP : %d.%d.%d.%d\n", local_ip[0], local_ip[1], local_ip[2], local_ip[3]);
+    	printf("Remote IP : %d.%d.%d.%d\n", remote_ip[0], remote_ip[1], remote_ip[2], remote_ip[3]);
```

Comment:
formatting: I think that a space after `Local IP` was to nicely align both addresses. Adding another one after `Remote IP` defeats the purpose ;)